.nvmrc

```diff
@@ -0,0 +1 @@
+16.3.0
```

Comment:
Encore une vielle habitude ...

J'aime bien utiilser [NVM](https://github.com/nvm-sh/nvm) pour gérer la version de node et de npm par projet.

Le fichier `.nvmrc` permet de dire à nvm "pour ce projet, utilise la version de node inscrite dans ce fichier"

Reply:
Je crois qu'il vaut mieux ne pas être trop dépendant des outils des uns et des autres.

Mais ça me fait dire qu'on n'a pas défini `engines.node` dans `package.json` : actuellement la nécessité d'utiliser Node v16 est donc informelle (seulement indiquée dans la doc). J'imagine qu'en plus `nvm` serait capable de lire cette info, non ? https://nodejs.dev/learn/the-package-json-guide#engines

Reply:
Si je résume ta proposition ce serait de mettre le fichier `.nvmrc`et mon `worklog.md` dans un `.gitignore` global ?

Si oui, pas d'inconvénient pour moi

Par rapport à 

>  On pourrait aussi défendre d'ajouter .nvmrc et .python-version pour remplacer les options nvm_node_version et pyenv_python_version dans le setup Ansible (ops/ansible/group_vars/web.yml)

Je ne connais pas vraiment ansible donc je ne peux émettre un avis. Cependant p-e un petit retour d'xp par rapport aux problèmes de gestion de version.

Le passage de node 14 à node 16 à été un peu (allez juste un peu) douloureux sur vimeet365 pcq pour passer de node 14 à node 16 cela impliquait d'être consistant

Cela veut dire que le projet, les dépendances du projet, les dockerFiles, les fichiers k8s, la conf de la CI, la conf de GCP, ...  prennent en compte (et autorise ....) ce changement de version.

Et bien sûr, tu n'as pas toujours à l'esprit qu'il faut aussi modifier tel ou tel fichier donc ça devient un peu casse pied.

Mon idéal serait de n'avoir à changer la version de node utilisée uniquement à 1 endroit pour que ce changement soit pris en compte partout.

P-e que nvm pourrait aider à ça ? On aura p-e l'occasion d'en parler plus en détail

Reply:
Pour l'instant je dirais qu'un tel changement de version est facilité par le fait qu'il n'y a "que" un setup local (informel pour l'instant, mieux formalisé si on définit `engines`), et un setup "live" entièrement défini dans `ops/` avec le fichier `ops/ansible/group_vars/web.yml`.